### ISSUE TITLE
Data parallel + zigzag_ring_attn support

### ISSUE THREAD

**Issue by WallE-Chang** [jzhang38/EasyContext#12]
Thanks for your excellent works！！
I found that if zigzag_ring_attn is used, data_parallel cannot take effect. It seems using all gpu to do seq parallel.

**Comment by jzhang38** [comment]
Yeah, I will definitely try to add this feature. So far I feel this is not trivial because I need to have some mechanism to manage the process group.

**Comment by nitinagarwal** [comment]
Is there any update on this?

**Comment by Charliiieeeeeee** [comment]
> Is there any update on this?

you can use xtuner to achieve dp + sequnece_parallel
